var/spack/repos/builtin/packages/emacs/package.py

```diff
@@ -79,4 +78,9 @@ def configure_args(self):
         if sys.platform == 'darwin':
             args.append('--without-ns')
 
+        if '+tls' in spec:
+            args = ['--with-gnutls']
```

Comment:
You'll need to use `append` here and below. Otherwise you're overwriting previous arguments.

Reply:
Will do.  Thanks.